### DISCUSSION TITLE
Google doesn't return auth_time

### DISCUSSION THREAD

**Discussion by bestickley** [panva/openid-client#584] [Q&A] [answered]
Hi,
I tried using this library to add sign in with Google to my app by following README [here](https://github.com/panva/node-openid-client/tree/main#authorization-code-flow). I see that Google doesn't return `auth_time` which this library throws an error about unless `require_auth_time` is set to `false` when constructing `Client`. Should the README example be updated?

**Comment by panva** [comment, marked as answer]
`require_auth_time` defaults to a falsy value which means you don't have to provide it and no `auth_time` is required to be returned by the IdP.

The example you link doesn't set `require_auth_time` to true. So it cannot throw `auth_time` missing errors.

**Reply by bestickley** [reply]
@panva, thank you for the response. Where in the code does `require_auth_time` default to falsy value? The only reference I see to "require_auth_time" within the lib folder is [this line](https://github.com/panva/node-openid-client/blob/main/lib/client.js#L747)? Am I missing something?

**Reply by panva** [reply]
The fact that it's not initialized to a particular value means when it's accessed its value is by default `undefined`, which is a falsy value.

**Reply by bestickley** [reply]
Thank you for explaining. Sorry for such a silly question.
